neuromation/cli/admin.py

```diff
@@ -26,4 +28,47 @@ def admin() -> None:
     )
 
 
+@command()
+@click.argument("cluster_name", required=True, type=str)
+@click.argument("user_name", required=True, type=str)
+@click.argument(
```

Comment:
That's interesting: `click` renders `ROLE` argument as mandatory according to README.

Reply:
Yea and if I remove metavar it prints bad error messages:

```
Invalid value for "[[admin|manager|user]]": invalid choice: my_role. (choose from admin, manager, user)
```

Don't know which is better

Reply:
What if set `metavar="[ROLE]"`?

Reply:
Yea, at least that looks better